test/e2e-common.sh

```diff
@@ -143,6 +143,11 @@ function test_task_creation() {
             all_status=$(kubectl get -n ${tns} pipelinerun --output=jsonpath='{.items[*].status.conditions[*].status}')
             reason=$(kubectl get -n ${tns} pipelinerun --output=jsonpath='{.items[*].status.conditions[*].reason}')
 
+            # once again check for the all_status in case it doesn't get's it for the first time
+            if [[ -z ${all_status} ]]; then
```

Comment:
I would prefer waiting for that to return something, there may be scenario even after one more check we get the empty status